sysdump/sysdump.go

```diff
@@ -679,6 +681,38 @@ func (c *Collector) Run() error {
 				return nil
 			},
 		},
+		{
+			Description: "Collecting the Cilium Envoy configuration",
+			Quick:       true,
+			Task: func(ctx context.Context) error {
+				v, err := c.Client.GetConfigMap(ctx, c.Options.CiliumNamespace, ciliumEnvoyConfigMapName, metav1.GetOptions{})
+				if err != nil {
+					return fmt.Errorf("failed to collect the Cilium Envoy configuration: %w", err)
```

Comment:
Again, since this is optional, this maybe shouldn't be an error if the ConfigMap isn't found?

Reply:
see https://github.com/cilium/cilium-cli/pull/1546#discussion_r1189425966